src/pages/datasetView/DatasetList.tsx

```diff
@@ -49,7 +49,7 @@ class CancerStudyCell extends React.Component<ICancerStudyCellProps,{}> {
         return (
             <span>
                 <a
-                    href={`http://www.cbioportal.org/study?id=${this.props.studyId}#summary`}
```

Comment:
@inodb @alisman: it seems this hard coded reference to www.cbioportal.org has been there for quite some time. At the same time, the datasets page seems to work correctly on a local installation. How is this possible? Is there a global search+replace going on during build time?

Reply:
look like you had an old version of this file.  In the below, you can see this was fixed.  On the other hand, there are other references to our datahub.  Not sure if those should also be configurable?

https://github.com/cBioPortal/cbioportal-frontend/blob/master/src/pages/datasetView/DatasetList.tsx

Reply:
indeed old code. Sorry for alarming you. Will make a new PR.